### DISCUSSION TITLE
Background Image VTK to Trame

### DISCUSSION THREAD

**Discussion by astav002** [Kitware/trame#198] [Q&A] [answered]
Hi,

I am trying to load a background image in VTK and display in Trame application.  I am basing the VTK portion on https://kitware.github.io/vtk-examples/site/Python/Images/BackgroundImage/ which works out of the box.  However, when I strip this down to just the background image for Trame.  Nothing displays except the superquadratic?  I could use the VImg to display an image, but I want to be able to interact and draw on the image in a new layer.  Any thoughts or guidance on this?

**Comment by jourdain** [comment, marked as answer]
If you are using VtkRemoteView, it should work.

If not, on the web side you could put an image under the view, but since the view is opaque, you won't be able to see it unless you set the background color of the renderer to be transparent.

**Reply by astav002** [reply]
Thanks @jourdain !  That did the trick!

**Reply by astav002** [reply]
To ask a follow up... I want to get a global reference with respect to the 2D background image on where a mouseclick occurred.  For example: let's say use clicked point (50, 50).  I can get the mouseclick event and the location, but if it is rescaled by the user then the location changes if they were to click the same spot again.  How would I go about either restricting scaling/modification of the background image, or is it best practice that I pull all of the scale, translation, skew etc. from the VTK frame to determine this?

**Reply by jourdain** [reply]
You should ask the server to do the lookup between display coordinates and world.
